packages/babel-plugin-emotion/__tests__/__fixtures__/core-with-component.js

```diff
@@ -0,0 +1,6 @@
+// @flow
+import styled from '@emotion/styled'
+
+const MyComponent = styled.div({ color: 'hotpink' })
+
+MyComponent.withComponent('section')
```

Comment:
It seems that assigning it to a variable was actually useful for tests because it made label inferrable, so probably we should bring it back - sorry for the confusion. I would just name it `OtherComponent` or smth similar

Reply:
🆗